easybuild/easyblocks/q/quantumespresso.py

```diff
@@ -262,7 +384,7 @@ def configure_step(self):
             raise EasyBuildError("Found FoX external module, QuantumESPRESSO" +
                                  "must use the version they include with the source.")
 
-        self.log.debug("List of replacements to perform: %s" % repls)
+        self.log.debug("List of replacements to perform: %s" % str(self.repls))
```

Comment:
```suggestion
        self.log.info("List of replacements to perform: %s" % str(self.repls))
```
Too important for a debug log